apps/site/assets/ts/stop/components/StopPageRedesign.tsx

```diff
@@ -56,16 +60,21 @@ const StopPageRedesign = ({
   // not just specific stops or trips
   const routeWideAlertsArray = routeWideAlerts(alertsForRoutesResult.data);
   // Get only alerts that are current
+  const alertsWithinSevenDays = filter(
+    alertsForStopResult.data.concat(routeWideAlertsArray),
```

Comment:
nit-pick: This could be set as its own variable (`allAlerts`) for reuse in the filter below as well.